sample/wikifile.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from pprint import pprint
+import json
+
+with open('metadata.json') as data_file:
+    data = json.load(data_file)
+
+
+class WikiFile:
+    def __init__(self, **reference_to_pages):
+        self.base_url = "https://en.wikipedia.org/w/api.php"
+
+        self.payload = {
+            'action': 'query',
+            'format': 'json',
+        }
+        self.headers = {
+                'User-Agent': data["name"] + ' (' + data["website"] + ')'
+        }
+
+        self._parse_kwargs(**reference_to_pages)
+
+    def _parse_kwargs(self, **kwparams):
+        if 'pageids' in kwparams:
+            self.payload['pageids'] = '|'.join(str(ID) for ID in list(kwparams['pageids']))
+        elif 'titles' in kwparams:
+            self.payload['titles'] = '|'.join(str(ID) for ID in list(kwparams['titles']))
```

Comment:
As we discussed, passing multiple titles will cause problems because all will return `-1` as `pageid`. Are you sure we should use `pageid` as the key in `df_list`?